### PR TITLE
Version Patch

### PR DESCRIPTION
**Pull request by cyrillkalita**
- Update dependency to support Laravel 8
- Add [CHANGELOG.md](CHANGELOG.md)
- Add composer scripts for testing and code fixing
- Add ignore of .php_cs.cache
- Apply PSR style formatting

### CODE REVIEW

**Comment by nickfls** [comment]
@rtconner Rob, is there anything stopping the merge?
If i took too much liberty in running PSR fixes etc, I can create another PR with just a composer dependency update

**Comment by rtconner** [comment]
just lack of time on my part. sorry. I didn't even see this PR. what are all the white space changes?

**Comment by rtconner** [comment]
oh you probably did reformatting with phpstorm. I see.

**Comment by cyrillkalita** [comment]
@rtconner I ran the code through phpcs.

**Comment by cyrillkalita** [comment]
@rtconner do you think you may be able to review it any time soon?

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ php artisan migrate
 
 ```php
 class Article extends \Illuminate\Database\Eloquent\Model {
-	use \Conner\Likeable\LikeableTrait;
+	use \Conner\Likeable\Likeable;
```
Comment (rtconner) [inline README.md:29]: wow, thanks for fixing this, I hadn't realized docs were wrong

**Review by rtconner — COMMENTED** [review]
Ok this is released as version 3.1.0. 
Thank you!

**Pull request merged**